cluster-autoscaler/core/autoscaler.go

```diff
@@ -101,8 +101,8 @@ func initializeDefaultOptions(opts *AutoscalerOptions) error {
 		opts.CloudProvider = cloudBuilder.NewCloudProvider(opts.AutoscalingOptions)
 	}
 	if opts.ExpanderStrategy == nil {
-		expanderStrategy, err := factory.ExpanderStrategyFromString(opts.ExpanderName,
-			opts.CloudProvider, opts.AutoscalingKubeClients, opts.KubeClient, opts.ConfigNamespace)
+		expanderStrategy, err := factory.ExpanderStrategyFromString(opts.ExpanderName, opts.FallbackExpanderName, opts.GRPCExpanderCert,
+			opts.GRPCExpanderURL, opts.CloudProvider, opts.AutoscalingKubeClients, opts.KubeClient, opts.ConfigNamespace)
```

Comment:
this wasn't super clean but idk if making a struct specifically for expander stuff would be much better?

Reply:
nah, it was ugly before, this doesn't really make it any uglier.  and it makes the PR smaller.